merge_streams_block.py

```diff
@@ -44,15 +44,34 @@ def process_group_signals(self, signals, group, input_id):
         if merged_signals:
             self.notify_signals(merged_signals)
 
+    def _helper_merge_signals(self, sig_1_dict, sig_2_dict):
+        ''' Recurively merge the signals passed into _merge_signals() '''
+        dict1_key = list(sig_1_dict.keys())[0]
+        dict2_key = list(sig_2_dict.keys())[0]
```

Comment:
What if the signals have more than one attribute/key? It looks to me like a more complex signal wouldn't work.